plugins/mpris2/mprisobject.py

```diff
@@ -217,13 +225,7 @@ def Identity(self):
     @property
     def SupportedMimeTypes(self):
         # Taken from exaile.desktop
-        mimetypes = 'audio/musepack;application/musepack;application/x-ape;' \
-            'audio/ape;audio/x-ape;audio/x-musepack;application/x-musepack;' \
-            'audio/x-mp3;application/x-id3;audio/mpeg;audio/x-mpeg;' \
-            'audio/x-mpeg-3;audio/mpeg3;audio/mp3;audio/x-m4a;audio/mpc;' \
-            'audio/x-mpc;audio/mp;audio/x-mp;application/ogg;' \
-            'application/x-ogg;audio/vorbis;audio/x-vorbis;audio/ogg;' \
-            'audio/x-ogg;audio/x-flac;application/x-flac;audio/flac'
+        mimetypes = 'audio/musepack;application/musepack;application/x-ape;' 'audio/ape;audio/x-ape;audio/x-musepack;application/x-musepack;' 'audio/x-mp3;application/x-id3;audio/mpeg;audio/x-mpeg;' 'audio/x-mpeg-3;audio/mpeg3;audio/mp3;audio/x-m4a;audio/mpc;' 'audio/x-mpc;audio/mp;audio/x-mp;application/ogg;' 'application/x-ogg;audio/vorbis;audio/x-vorbis;audio/ogg;' 'audio/x-ogg;audio/x-flac;application/x-flac;audio/flac'
```

Comment:
This line is way too long which does not make any sense to me. Can we prevent that from happening?

Reply:
Looks like a bug. Filed upstream at https://github.com/ambv/black/issues/330

Reply:
Also affects `Gtk.Template` stuff as noted above. See https://github.com/ambv/black/issues/330#issuecomment-395995308 for an example.